CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 ## 6.6.0 (unreleased)
 
-- Adapt Dockerfile to the removal of psycopg2-binary
-- DOCKER: Added env `POSTGRES_SSLMODE` with default value disabled (thanks to [@ribeiromiranda](https://github.com/ribeiromiranda))
+- Add support for Python 3.11
+- Remove support for Python 3.7
+- Upgrade to Django 3.2 (and upgrade some other dependencies)
```

Comment:
did this remove support to some Django versions?

Reply:
Taiga always guarantees compatibility with a single version of Django (defined in the requirements.txt). 

Could it be compatible with versions higher than this version? Yes, but I haven't tried it. And with some minor? Maybe, it will depend on if there are  broken changes or incompatibilities with third party libraries